cli/cmd/root.go

```diff
@@ -81,27 +82,20 @@ func initConfig() {
 		logging.LogLevel = logging.QuietLevel
 	}
 
+	cfgMgr := config.NewCLIConfigManager()
```

Comment:
I have used `CLIConfigManager` here instead of `viper` because we already have it in place and it works well. 
TODO for the future can be to re-write `NewCLIConfigManager` with viper.